bam_masterdata/checker/masterdata_validator.py

```diff
@@ -1,7 +1,7 @@
-import json
 import re
 
 from bam_masterdata.logger import logger
+from bam_masterdata.metadata.definitions import DataType
 from bam_masterdata.utils import is_reduced_version
 
 

```

Comment:
a small typo (I think): `class MasterdataValidator` (now `Data` is capitalize)

Furthermore, I have a similar suggestion as with the `MasterdataChecker` class. Instead of storing in the class mem the dictionaries, simply return then in the corresponding function.

I saw that you are actually returning, but still storing in `self` variables. I would simply reject this if these are not use somewhere else.